e2e_tests/integration/caseSummary.feature

```diff
@@ -11,8 +11,8 @@ Feature: Case summary
     And Maria views the Contact history page
     And Maria filters contacts by date range
     And Maria starts a new recommendation
-    And Maria recommends a fixed term recall
-    And Maria selects a custody status
-    Then Maria sees a confirmation page
-    And Maria downloads the Part A
-    And Maria updates the recommendation
\ No newline at end of file
+#    And Maria recommends a fixed term recall
```

Comment:
You mean to comment this out?

Reply:
Yes, the E2E will fail until your PR is in as well, then I'll re-enable them